sklearn/neural_network/tests/test_mlp.py

```diff
@@ -177,7 +177,8 @@ def test_gradient():
     for n_labels in [2, 3]:
         n_samples = 5
         n_features = 10
-        X = np.random.random((n_samples, n_features))
```

Comment:
I think you've not merged changes from master correctly.

As described at https://scikit-learn.org/dev/developers/contributing.html, you should use
```bash
$ git fetch upstream
$ git merge upstream/master
```

Reply:
Thanks @jnothman - using those commands I get :

```bash
$ git fetch upstream
$ git merge upstream/master
Already up to date.
```

Reply:
These changes you are seeing might be related to @hermidalc work on #13470